lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -3218,8 +3218,12 @@ public String getReleaseString(SimplePageItem i) {
 		     return null;
 		 if (page.isHidden())
 		     return messageLocator.getMessage("simplepage.hiddenpage");
-		 if (page.getReleaseDate() != null && page.getReleaseDate().after(new Date()))
-		     return messageLocator.getMessage("simplepage.pagenotreleased");
+		 if (page.getReleaseDate() != null && page.getReleaseDate().after(new Date())) {
+			 Date releaseDate = page.getReleaseDate();
+			 SimpleDateFormat dateFormat = new SimpleDateFormat("EEE M/d/yy");
```

Comment:
The rest of the world doesn't use M/d ... is there some magic internationalization happening anywhere here?